es-common-extensions/src/security/securityRequests.tsx

```diff
@@ -1,6 +1,6 @@
 
 const mode = import.meta.env.NODE_ENV
-const securityUrl = import.meta.env[`ES_SECURITY_URL_${mode}`]
+const securityUrl = import.meta.env[`VITE_ES_SECURITY_URL_${mode}`]
```

Comment:
vite requires all env variables to have prefix of "VITE_" if they are to be available for use in the code. should have warning message for users who have not updated their code yet?